lib/kafka_gen_stage/consumer.ex

```diff
@@ -141,7 +141,7 @@ defmodule KafkaGenStage.Consumer do
           consumer: pid(),
           consumer_ref: reference(),
           queue: :queue.queue(),
-          high_wm_offset: non_neg_integer() | atom(),
+          is_end_of_stream: boolean(),
```

Comment:
The type of  bulk_transformer is still `([msg_tuple] -> [msg_tuple])` (cannot make a comment at that line, it's on line 91).